.vscode/launch.json

```diff
@@ -56,6 +56,19 @@
             "svdFile": "${workspaceRoot}/firmware/ext/cmsis-svd/data/STMicro/STM32F427.svd",
             "servertype": "openocd",
             "configFiles": ["interface/stlink.cfg", "target/stm32f4x.cfg"],
+            "runToMain": true
+		},
+		{
+            "name": "Debug F7x6",
+            "type": "cortex-debug",
```

Comment:
is this change supposed to be in this PR?

Reply:
I left it there since i used it for debugging and we had no support for f7, i ca make it in a new pr, just wanted to reduce you're load and left it there

Reply:
oh if that's useful stuff let's keep it here